pyspecProcScripts/load_data.py

```diff
@@ -599,6 +599,8 @@ def proc_spincore_ODNP_v4(s, fl=None):
 
 
 def proc_spincore_generalproc_v1(s, fl=None):
+    if "tau_us" in s.get_prop("acq_params").keys():
+        s["t2"] -= s.get_prop("acq_params")["tau_us"] * 1e-6
```

Comment:
this is a nice way of doing this. Should I extend this method into the load_data PR for subtracting tau?